.github/workflows/wheels.yml

```diff
@@ -125,7 +124,26 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.9'  # update once build dependencies are available
+          python-version: "3.11" # update once build dependencies are available
+
+      - name: Install conda for macos arm64
+        if: ${{ matrix.platform_id == 'macosx_arm64' }}
+        run: |
+          # macos arm64 runners do not have conda installed. Thus we much install conda manually
+          MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/download/23.11.0-0/Mambaforge-23.11.0-0-MacOSX-arm64.sh"
```

Comment:
Please also include the expected sha256 of this installer and check the match with sha256sum.